### PR TITLE
feat: intave speed

### PR DESCRIPTION
**Pull request by larryngton2**
- lowhop (0.42 - 1.7e-14)
- airboost (1.003)
- groundstrafe (0.29 strength)

### CODE REVIEW

**Comment by TheColorfulGabibbo** [comment]
add 45° yaw, makes the speed faster

**Comment by larryngton2** [comment]
> add 45° yaw, makes the speed faster

might flag with boost, I'll try though

**Comment by TheColorfulGabibbo** [comment]
so? does it work?

**Comment by Husky272** [comment]
Wait，u sure that 0.42f - 1.7E-14f is not equal 0.42f？

**Comment by larryngton2** [comment]
> Wait，u sure that 0.42f - 1.7E-14f is not equal 0.42f？

1.7e-14 is 0.000000000000007 idk if kotlin rounds it

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/intave/SpeedIntave14.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+class SpeedIntave14(override val parent: ChoiceConfigurable<*>) : SpeedBHopBase("Intave14", parent) {
+
+    private class Strafe(parent: Listenable?) : ToggleableConfigurable(parent, "Strafe", true) {
+        private val strength by float("Strength", 0.29f, 0.01f..0.29f)
+
+        @Suppress("unused")
+        val moveHandler = handler<PlayerMoveEvent> { event ->
+            if (event.type == MovementType.SELF) {
+
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/intave/SpeedIntave14.kt]: remove the weird space

**Comment by larryngton2** [comment]
sorry i was sleeping at that time

**Pull request merged**